includes/polling/applications/nvidia.inc.php

```diff
@@ -8,10 +8,18 @@
 $options      = '-O qv';
 $oid          = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.6.110.118.105.100.105.97';
 $gpus = snmp_walk($device, $oid, $options);
-update_application($app, $gpus);
-
 $gpuArray = explode("\n", $gpus);
 
+$gpus=0;
+$sm_total=0;
+while (isset($gpuArray[$gpus])) {
+    $sm_total = $sm_total+explode(",", $gpuArray[$gpus])[3];
```

Comment:
You can do:
```
foreach ($gpuArray as $gpu) {
    $stats = explode(",", $gpu);
    $sm_total += $stats[3];
}
$sm_average = ($sm_total ? 0 : ($sm_total / count($gpuArray));
```

Can't reference an array like that in php 5.3 :-)
Need to avoid divide by zero too.